src/applications/vaos/tests/e2e/community-care.cypress.spec.js

```diff
@@ -0,0 +1,186 @@
+import { initAppointmentListMock } from './vaos-cypress-helpers';
+
+describe('Create new community care appointment', () => {
+  beforeEach(() => {
+    initAppointmentListMock();
+    // Stub out Google Analytics call
+    cy.route({
```

Comment:
I don't think you need to mock these GA or SOCKS calls, though I don't think it hurts anything either.